### ISSUE TITLE
How about use `isomorphic-ws` for nodejs client

### ISSUE THREAD

**Issue by foreseaz** [radarsu/rpc-websocket-client#1]
* **I'm submitting a ...**

- [x] feature request

* **Summary**
Propose to use `isomorphic-ws` instead of native [WebSocket](https://github.com/radarsu/rpc-websocket-client/blob/master/src/lib/index.ts#L103).

* **Error trace**
```
ReferenceError: WebSocket is not defined
    at RpcWebSocketClient.connect (foo/bar/node_modules/rpc-websocket-client/src/lib/index.ts:102:23)
```

**Comment by foreseaz** [comment]
Could be much more convenient for node tests

**Comment by radarsu** [comment]
I'll add customization option to replace native WebSocket.

**Comment by foreseaz** [comment]
yep, I tried that out but not work, because TS check the `WebSocket` type during compile time.

**Comment by radarsu** [comment]
Published version 1.0.3 which uses isomorphic-ws. Needs tests though. I'll try to write some later on.
